server/src/game/end_game_condition.rs

```diff
@@ -2,26 +2,113 @@ use super::{role_list::Faction, role::Role};
 
 #[derive(Debug, PartialEq, Eq, Clone, Hash)]
 pub enum EndGameCondition {
-    Faction(Faction),
-    Role(Role),
+    Town,
+    Mafia,
+    Cult,
 
-    None
+    Arsonist,
+    Werewolf,
+    Ojo,
+
+    Death,
+    Politician,
+
+    Draw
 }
 impl EndGameCondition {
```

Comment:
It really seems like now, this represents a "win condition", am I correct?

Reply:
idk, Draw isnt a win condition. Its the state of the game when it ends